lib/bundle_update_interactive/cli/table.rb

```diff
@@ -28,7 +28,7 @@ def render_gem(name)
 
     def render
       lines = [render_header]
-      rows.each_key { |name| lines << render_gem(name) }
+      rows.keys.sort.each { |name| lines << render_gem(name) }
```

Comment:
🗒️ I moved sorting behavior into `CLI::Table`, out of the core `Report` class. This makes the report implementation simpler, and sorting is naturally more of a UI (CLI in our case) concern.